ssh/config.go

```diff
@@ -42,12 +42,13 @@ func AddHost(alias, host, user string, port int) error {
 		return err
 	}
 
-	home, err := os.UserHomeDir()
-	if err != nil {
-		ui.Errorf("Failed to get user home directory: %v", err)
-		os.Exit(1)
+	// Add an Include directive to the user's ssh config to unweave_global SSH configs - used for vscode-remote.
+	// Create one it if it does not exist.
+	if _, err := os.Stat(sshConfigPath); os.IsNotExist(err) {
```

Comment:
This should also check for directories/sub-directories and not just the file.

Reply:
Yeah exactly. In this case you're passing the path as a variable which could be arbitrarily deeply nester. So we need a `os.MkdirAll`

Reply:
Great thanks for putting the time in to clarify!

Reply:
I've added a MkDir check using the same permissions as the .ssh folder